src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/transform/resthandler/RestIndexTransformAction.kt

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexmanagement.transform.resthandler
+
+import com.amazon.opendistroforelasticsearch.indexmanagement.IndexManagementPlugin.Companion.TRANSFORM_BASE_URI
+import com.amazon.opendistroforelasticsearch.indexmanagement.elasticapi.parseWithType
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.index.IndexTransformAction
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.index.IndexTransformRequest
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.action.index.IndexTransformResponse
+import com.amazon.opendistroforelasticsearch.indexmanagement.transform.model.Transform
+import com.amazon.opendistroforelasticsearch.indexmanagement.util.IF_PRIMARY_TERM
+import com.amazon.opendistroforelasticsearch.indexmanagement.util.IF_SEQ_NO
+import com.amazon.opendistroforelasticsearch.indexmanagement.util.REFRESH
+import org.elasticsearch.action.support.WriteRequest
+import org.elasticsearch.client.node.NodeClient
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.index.seqno.SequenceNumbers
+import org.elasticsearch.rest.*
```

Comment:
minor: avoid using wild card import, replaces with the exact one

Reply:
Fixed, IntellJ was collapsing them.